build/cpu1/exe/cfe_es_startup.scr

```diff
@@ -1,28 +1,10 @@
-CFE_LIB, /cf/cfs_lib.so,      CFS_LibInit,      CFS_LIB,        0,       0, 0x0, 0;
-CFE_LIB, /cf/osk_app_fw.so,  OSK_APP_FwInit,   OSK_APP_FW,     0,    8192, 0x0, 0;
-CFE_LIB, /cf/expat_lib.so,    EXPAT_Init,       EXPAT_LIB,      0,    8192, 0x0, 0;
-CFE_APP, /cf/kit_to.so,       KIT_TO_AppMain,   KIT_TO,        70,   16384, 0x0, 0;
-CFE_APP, /cf/kit_ci.so,       KIT_CI_AppMain,   KIT_CI,        60,   16384, 0x0, 0;
-CFE_APP, /cf/kit_sch.so,      KIT_SCH_AppMain,  KIT_SCH,       50,   16384, 0x0, 0;
-CFE_APP, /cf/ds.so,           DS_AppMain,       DS,            80,   16384, 0x0, 0;
-CFE_APP, /cf/fm.so,           FM_AppMain,       FM,            80,   16384, 0x0, 0;
-CFE_APP, /cf/lc.so,           LC_AppMain,       LC,            80,   16384, 0x0, 0;
-CFE_APP, /cf/sc.so,           SC_AppMain,       SC,            80,   16384, 0x0, 0;
-CFE_APP, /cf/bm.so,           BM_AppMain,       BM,            80,   16384, 0x0, 0;
-CFE_APP, /cf/mm.so,           MM_AppMain,       MM,            90,   16384, 0x0, 0;
-CFE_APP, /cf/hc.so,           HC_AppMain,       HC,            80,   16384, 0x0, 0;
-CFE_APP, /cf/sim.so,         SIM_AppMain,      HSIM,          90,   16384, 0x0, 0;
-CFE_APP, /cf/f42.so,          F42_AppMain,      F42,           90,   16384, 0x0, 0;
-CFE_APP, /cf/i42.so,          I42_AppMain,      I42,           90,   80192, 0x0, 0;
-!CFE_APP, /cf/cs.so,           CS_AppMain,       CS,            90,   16384, 0x0, 0;
-!CFE_APP, /cf/md.so,           MD_AppMain,       MD,            90,   16384, 0x0, 0;
-!CFE_APP, /cf/tftp.so,         TFTP_AppMain,     TFTP,         100,   16384, 0x0, 0;
-!CFE_APP, /cf/osk_demo_app.so, OSK_DEMO_AppMain, OSK_DEMO,     120,    8192, 0x0, 0;
-!CFE_APP, /cf/hs.so,           HS_AppMain,       HS,            80,   16384, 0x0, 0;
-!CFE_APP, /cf/example.so,      EXAMPLE_AppMain,  EXAMPLE,       90,   16384, 0x0, 0;
-!CFE_LIB, /cf/sample_lib.so,  SAMPLE_LibInit,  SAMPLE_LIB,    0,      0,  0x0, 0;
-!CFE_APP,cfs /cf/sample_app.so,  SAMPLE_AppMain,  SAMPLE_APP,  150,   16384, 0x0, 0;
-!CFE_LIB, /cf/math_lib.so,    Math_Init,       MATH_LIB,      0,  8192,  0x0, 0;
+CFS_LIB, /cf/apps/cfs_lib.so,     CFS_LibInit,     CFS_LIB,       0,   0,    0x0, 0;
+CFE_APP, /cf/apps/sch_lab.so,     SCH_Lab_AppMain, SCH_LAB_APP,  80,   8192, 0x0, 0;
+CFE_APP, /cf/apps/hs.so,          HS_AppMain,      HS,           81,   8192, 0x0, 0;
+CFE_APP, /cf/apps/fm.so,          FM_AppMain,      FM,           82,   8192, 0x0, 0;
+CFE_APP, /cf/apps/sc.so,          SC_AppMain,      SC,           83,   8192, 0x0, 0;
+CFE_APP, /cf/apps/ds.so,          DS_AppMain,      DS,           84,   8192, 0x0, 0;
+CFE_APP, /cf/apps/mm.so,          MM_AppMain,      MM,           85,   8192, 0x0, 0;

```

Comment:
Can you double check that this matches what components we actually have on the composite side?

Reply:
I might have missed `hs`, which is in `llboot_cFE.sh`

Reply:
Does the system boot if you add it?

Reply:
It's in line 3 actually, just doesn't show up in preview